probe/appclient/probe_config.go

```diff
@@ -46,6 +46,9 @@ func (pc ProbeConfig) authorizeHeaders(headers http.Header) {
 	}
 	headers.Set(xfer.ScopeProbeIDHeader, pc.ProbeID)
 	headers.Set(xfer.ScopeProbeVersionHeader, pc.ProbeVersion)
+	headers.Set("user-agent","Probe_Scope/"+pc.ProbeVersion )
```

Comment:
Could you please rename like this as @bboreham suggested?

```suggestion
	headers.Set("user-agent","Scope_Probe/"+pc.ProbeVersion )
```

It's also more consistent with the rest of the code.

Reply:
Aaah. I thought i got it the right way the first time itself. Will commit this change.